easybuild/easyconfigs/r/rootpy/rootpy-0.8.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'rootpy'
+version = '0.8.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.rootpy.org/'
+
+description = """The rootpy project is a community-driven initiative aiming to provide a more pythonic interface with 
+ROOT on top of the existing PyROOT bindings. Given Python’s reflective and dynamic nature, rootpy also aims to 
+improve ROOT design flaws and supplement existing ROOT functionality. The scientific Python community also offers a 
+multitude of powerful packages such as SciPy, NumPy, matplotlib, scikit-learn, and PyTables, but a suitable interface 
+between them and ROOT has been lacking. rootpy provides the interfaces and conversion mechanisms required to liberate 
+your data and to take advantage of these alternatives if needed."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/rootpy/%(namelower)s/archive/']
+sources = ['%(version)s.tar.gz']
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('ROOT','v6.08.02','-Python-%(pyver)s'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s')
+]
+
+options = {'modulename': 'os'}
+
+#sanity_check_paths = {
+#    'files': ['bin/quast.py'],
+#    'dirs': [''],
+#}
```

Comment:
@ysagon This should be fixed, to something like:

```python
sanity_check_paths = {
    'files': [],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```

of course, it other important files are also installed, they can be listed here too